observatorium/overlays/moc/smaug/thanos/deployments/observatorium-thanos-query.yaml

```diff
@@ -0,0 +1,99 @@
+kind: Deployment
```

Comment:
this already exists
https://github.com/operate-first/apps/blob/master/observatorium/base/instance/thanos/query/deployment.yaml